projects/opendr_ws/src/perception/scripts/landmark_based_facial_expression_recognition.py

```diff
@@ -134,16 +134,40 @@ def _landmark2numpy(landmarks):
 
 
 if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input_image_topic", help="Topic name for input image",
+                        type=str, default="/usb_cam/image_raw")
+    parser.add_argument("-o", "--output_category_topic", help="Topic name for output recognized category",
+                        type=str, default="/opendr/landmark_expression_recognition")
+    parser.add_argument("-d", "--output_category_description_topic", help="Topic name for category description",
+                        type=str, default="/opendr/landmark_expression_recognition_description")
+    parser.add_argument("--device", help="Device to use, either \"cpu\" or \"cuda\", defaults to \"cuda\"",
+                        type=str, default="cuda", choices=["cuda", "cpu"])
+    parser.add_argument("--model", help="Model to use, either 'pstbln_ck+', 'pstbln_casia', 'pstbln_afew'",
+                        type=str, default="pstbln_afew", choices=['pstbln_ck+', 'pstbln_casia', 'pstbln_afew'])
+    parser.add_argument("-s", "--shape_predictor", help="Shape predictor (landmark_extractor) to use",
+                        type=str, default='./predictor_path')
+    args = parser.parse_args()
+
     # Select the device for running the
     try:
-        if torch.cuda.is_available():
-            print("GPU found.")
-            device = 'cuda'
-        else:
+        if args.device == "cuda" and torch.cuda.is_available():
+            device = "cuda"
+        elif args.device == "cuda":
             print("GPU not found. Using CPU instead.")
-            device = 'cpu'
+            device = "cpu"
+        else:
+            print("Using CPU.")
+            device = "cpu"
     except:
-        device = 'cpu'
-
-    pose_estimation_node = LandmarkFacialExpressionRecognitionNode(device=device)
-    pose_estimation_node.listen()
+        print("Using CPU.")
+        device = "cpu"
+
+    landmark_expression_estimation_node = LandmarkFacialExpressionRecognitionNode(input_image_topic=args.input_image_topic,
+                                                                   output_category_topic=args.output_category_topic,
+                                                                   output_category_description_topic=
+                                                                   args.output_category_description_topic,
+                                                                   device=device, model=args.model,
+                                                                   shape_predictor=args.shape_predictor)
```

Comment:
There are some issues with indentation that fail the tests, the suggestion should fix it.

```suggestion
    landmark_expression_estimation_node =     landmark_expression_estimation_node = \
        LandmarkFacialExpressionRecognitionNode(input_image_topic=args.input_image_topic,
                                                output_category_topic=args.output_category_topic,
                                                output_category_description_topic=
                                                args.output_category_description_topic,
                                                device=device, model=args.model,
                                                shape_predictor=args.shape_predictor)
```